core/services/relay/evm/evm.go

```diff
@@ -159,14 +159,18 @@ func (r *Relayer) HealthReport() (report map[string]error) {
 }
 
 func (r *Relayer) NewPluginProvider(rargs commontypes.RelayArgs, pargs commontypes.PluginArgs) (commontypes.PluginProvider, error) {
+
+	// commontypes.Relayer doesn't wire context through
+	ctx := context.Background()
```

Comment:
These are not great, but my changes fix it and are coming soon, so maybe OK for this PR. You could mark them as:
```suggestion
	// TODO https://smartcontract-it.atlassian.net/browse/BCF-2887
	ctx := context.Background()
```